src/ceph-volume/plugin/zfs/setup.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = [ ]
+
+setup_requirements = [ ]
+
+test_requirements = [ ]
+
+setup(
+    author="Willem Jan Withagen",
+    author_email='wjw@digiware.nl',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Natural Language :: English',
+        "Programming Language :: Python :: 2",
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+    ],
+    description="Manage Ceph OSDs on ZFS pool/volume/filesystem",
+    install_requires=requirements,
+    license="BSD license",
+    long_description=readme + '\n\n' + history,
+    include_package_data=True,
+    keywords='ceph_volume_zfs',
+    name='ceph_volume_zfs',
```

Comment:
i'd suggest follow the naming convention. so it would be `ceph-volume-zfs`. and do we really want to have a separated tool? is `ceph-volume` able to call `ceph-volume-zfs` instead ?